scripts/load_mae_vit.py

```diff
@@ -0,0 +1,13 @@
+from collections import OrderedDict
```

Comment:
@anwai98 I have added this script as example for how to load the MAE checkpoint. I will share the link to the checkpoint via RocketChat.

Reply:
Hi @constantinpape @cdalinghaus. I found the issue, and went ahead and fixed it (https://github.com/constantinpape/torch-em/pull/182).

In short, the ViT is inherited from `timm`, which builds an end-to-end vision transformer for downstream tasks (hence, has the `head` component at the end). The original MAE adaptation just [discards this layer](https://github.com/facebookresearch/mae/blob/main/main_finetune.py#L233-L242), hence I tried here the same and it works. I made the checkpoint more functional to just accept the MAE weights directly (just passing the path to the pretrained MAE weights should do the trick atm). Thanks!